.github/workflows/conformance-aws-cni-v1.11.yaml

```diff
@@ -123,11 +123,11 @@ jobs:
     runs-on: ubuntu-latest
     timeout-minutes: 45
     steps:
-      - name: Checkout master branch to access local actions
+      - name: Checkout PR to access local actions
         uses: actions/checkout@24cb9080177205b6e8c946b17badbe402adc938f # v3.4.0
         with:
-          ref: ${{ github.event.repository.default_branch }}
```

Comment:
Can you explain why that is required? I'm concerned over the potential security impact of this change since we are now checking out untrusted user code.

Reply:
This was needed to actually test my changes to the CI workflows, otherwise it would fail to checkout the setup-eks-cluster action I created.

Beyond that, I'm not sure if this change makes sense beyond testing. I had thought that since we only run these CI jobs via issue comments it would be safe, but I realize that anyone can make these comments on their own PRs, so that change should probably be a "DO NOT MERGE" commit. I can make sure to remove these changes before we merge the PR.

Reply:
Yes, the reason we have this step is to protect against a malicious user checking out their malicious PR in the CI.